src/main/java/org/jenkinsci/plugins/tuleap_git_branch_source/config/TuleapConnector.java

```diff
@@ -65,53 +68,46 @@ public static FormValidation checkCredentials(@AncestorInPath Item item, String
-                CredentialsMatchers.allOf(withId(credentialsId), allUsernamePasswordMatch()));
-            try {
-                Boolean credentialsAreValid = TuleapClientCommandConfigurer.<Boolean>newInstance(
-                    defaultIfEmpty(apiUri, TuleapConfiguration.get().getApiBaseUrl()))
-                    .withCredentials(cred)
-                    .withCommand(new TuleapClientRawCmd.IsCredentialsValid())
-                    .configure()
-                    .call();
+                CredentialsMatchers.allOf(withId(credentialsId), allTuleapAccessTokenMatch()));
 
-                if (credentialsAreValid) {
-                    return FormValidation.ok();
-                } else {
-                    return FormValidation.error("Failed to validate the provided credentials");
-                }
-            } catch (IOException | IllegalArgumentException e) {
-                return FormValidation.error(e, "Failed to validate the provided credentials");
+            try {
+                checker.verifyAccessKey(Objects.requireNonNull(token).getToken());
+                return FormValidation.ok();
+            } catch (InvalidAccessKeyException | InvalidScopesForAccessKeyException exception) {
+                return FormValidation.error("Failed to validate the provided credentials");
             }
         }
     }
 
     @CheckForNull
-    public static StandardCredentials lookupScanCredentials(@CheckForNull Item context, @CheckForNull String apiUri,
+    public static TuleapAccessToken lookupScanCredentials(@CheckForNull Item context, @CheckForNull String apiUri,
         @CheckForNull String scanCredentialsId) {
         if (Util.fixEmpty(scanCredentialsId) == null) {
             return null;
         } else {
             return CredentialsMatchers
                 .firstOrNull(
-                    CredentialsProvider.lookupCredentials(StandardUsernameCredentials.class, context,
+                    CredentialsProvider.lookupCredentials(TuleapAccessToken.class, context,
                         context instanceof Queue.Task ? ((Queue.Task) context).getDefaultAuthentication()
                             : ACL.SYSTEM,
                         OFDomainRequirements(apiUri)),
                     CredentialsMatchers.allOf(withId(scanCredentialsId),
-                                              allUsernamePasswordMatch()));
+                                              allTuleapAccessTokenMatch()));
         }
     }
 
     private static List<DomainRequirement> OFDomainRequirements(@CheckForNull String apiUri) {
         return URIRequirementBuilder.fromUri(defaultIfEmpty(apiUri, TuleapConfiguration.get().getApiBaseUrl())).build();
     }
 
-    public static CredentialsMatcher allUsernamePasswordMatch() {
-        return CredentialsMatchers.anyOf(CredentialsMatchers.instanceOf(StandardUsernamePasswordCredentials.class));
+    public static CredentialsMatcher allTuleapAccessTokenMatch() {
```

Comment:
Is it possible to change the scope to private? It seems this method is used only in this class